pimpmycause/pimpmycause/templates/profiles/detail.html

```diff
@@ -0,0 +1,87 @@
+{% extends "layout.html" %}
+
+{% block title %} Profile {% endblock %}
+
+{% load static %}
+
+    {% block main %}
+    <section class="page-register pnl row">
+        <div class="content-wrap">
+            <h1 class="pnl__title">Profile</h1>
+                <div class="medium-12 large-12 small-10 large-centered small-centered columns">
+                        <div class="row">
+                            <div class="large-3 medium-4 small-10 small-centered medium-uncentered columns">
+
+                                {% if not profile_update_form.image.value %}
+                                    <img class="profile-img" src="https://encrypted-tbn1.gstatic.com/images?q=tbn:ANd9GcSwY0Ifx0mEN9C7n50Ub-viD3SQXUSrgYcNVu_nTrCRTpYcSGM9AQ-ZH0o" alt="Profile Image">
+
+                                {% else %}
+                                    <img class="profile-img" src="{{profile_update_form.image.value}}" alt="{{ user.name }} {{ user.surname }} Profile Image">
+                                {% endif %}
+                            </div>
+                            <div class="large-offset-3 medium-offset-2 large-4 medium-6 small-12 small-centered medium-uncentered columns">
+                                <label for="name"> Name </label>
+                                <input type="text" name="name" placeholder="Sam" value='{{ profile_update_form.name.value|default_if_none:"" }}' required>
```

Comment:
okay so I keep labels.. what will be the value of the p tags? For example in the one above it will be : 
<label for="name"> Name </label>
<p >{{ profile_update_form.name.value|default_if_none:"" }}</p>

or <p >{{person.name }}</p> (person comes from my view)